apps/src/lab2/lab2Redux.ts

```diff
@@ -89,6 +90,7 @@ const initialState: LabState = {
   levelProperties: undefined,
   isShareView: undefined,
   overrideValidations: undefined,
+  textToSpeechHasVoices: false,
```

Comment:
This seems like a good candidate to pull out into a separate redux file at some point, since it's not necessarily lab2-specific, right? Not blocking though, can happen later

Reply:
Correct.  This decision probably relates to the discussion above about whether there's any state that needs to be shared between the separate `TextToSpeech` UI components.